src/Nox.Types.Abstractions/TypeOptions/EnumerationTypeOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+using Nox.Types.Schema;
+using System.Collections.Generic;
+
+namespace Nox.Types;
+
+[Title("Options for the enum value type.")]
+[Description("Options to control the behaviour of the Enum nox type.")]
+[AdditionalProperties(false)]
+public class EnumerationTypeOptions : INoxTypeOptions
+{
+    [Title("The optional default values for this Enum.")]
+    [Description("The default evalues of this Enum. These values will automatically be created and available in the domain.")]
+    public IReadOnlyList<EnumerationValues>? Values { get; set; } 
```

Comment:
I would prefer to have an Empty Collection as the default value, and avoid having a nullable property, and therefore avoid unnecessary null checks
```suggestion
    public IReadOnlyList<EnumerationValues>? Values { get; set; }  = Array.Empty<EnumerationValues>();
```